### PR TITLE
jfleg dataset

### PR DESCRIPTION
**Pull request by debajyotidatta**
The dataset is for grammatical error correction.

https://huggingface.co/datasets/viewer/?dataset=jfleg

It is worth noting that the dataset also a benchmark for developing and evaluating systems with respect to fluency (the extent to which a text is native-sounding). This knowledge has been used to develop some of the prompts.

### CODE REVIEW

File: templates/jfleg/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      {{sentence}}
+
+
+      A correct version of the same sentence could be:
+
+
+      |||
+
+
+      {{ corrections | choice }}'
```
Comment (VictorSanh) [inline templates/jfleg/templates.yaml:89]: love the use of choice!

File: templates/jfleg/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
+dataset: jfleg
+templates:
+  13ada088-1e08-4c18-8042-50da516cd8b2: !Template
+    id: 13ada088-1e08-4c18-8042-50da516cd8b2
+    jinja: "{{sentence}}\n\nThe above sentence is hard to read.\n\nFour possible correct\
+      \ options are:\n\n|||\n\n{{ corrections[0]}} \n\nor\n\n{{ corrections[1]}} \n\
+      \nor\n\n{{ corrections[2]}}\n\nor\n\n{{ corrections[3]}} "
+    name: template_v2
+    reference: ''
+    task_template: true
```
Comment (VictorSanh) [inline templates/jfleg/templates.yaml:10]: i would remove this one
1/ it doesn't much stronger signal compared to v1
2/ you have to deal with possibly repetitive options
Capture d’écran 2021-06-14 à 3 33 35 PM

File: templates/jfleg/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    reference: Correcting the grammar
+    task_template: true
+  e9d603d4-cf1a-404e-8d82-cd29a04e82d3: !Template
+    id: e9d603d4-cf1a-404e-8d82-cd29a04e82d3
+    jinja: "The sentence below has grammatical errors that make it hard to read.\n\
+      \n{{sentence}}\n\nTwo possible ways of correcting the above sentence are:\n\n\
+      |||\n\n{{ corrections[0]}} \n\nand\n\n{{ corrections[1]}} "
+    name: template_v6
+    reference: ''
+    task_template: true
```
Comment (VictorSanh) [inline templates/jfleg/templates.yaml:100]: same comment as for _v2

Comment (VictorSanh) [reply templates/jfleg/templates.yaml:100]: + _v7

**Review by VictorSanh — CHANGES_REQUESTED** [review]
See my comments

a funky one that you can add is something like:
```jinja
I am correcting the grammar exercises of my students. How should be the following sentence re-written?
{{sentence}}|||
{{corrections | choice}}
```

**Comment by debajyotidatta** [comment]
> See my comments
> 
> a funky one that you can add is something like:
> 
> ```
> I am correcting the grammar exercises of my students. How should be the following sentence re-written?
> {{sentence}}|||
> {{corrections | choice}}
> ```

Added significant changes to the other prompt based on this suggestion. This adds variety and hopefully a much stronger signal.

**Comment by VictorSanh** [comment]
thank you @debajyotidatta, merging now, please update the spreadsheet!

**Pull request merged**